core/src/main/java/de/bwaldvogel/mongo/backend/aggregation/Aggregation.java

```diff
@@ -12,20 +12,7 @@
 import de.bwaldvogel.mongo.MongoCollection;
 import de.bwaldvogel.mongo.MongoDatabase;
 import de.bwaldvogel.mongo.backend.CollectionUtils;
-import de.bwaldvogel.mongo.backend.aggregation.stage.AddFieldsStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.AggregationStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.BucketStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.FacetStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.GroupStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.LimitStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.LookupStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.LookupWithPipelineStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.MatchStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.OrderByStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.ProjectStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.ReplaceRootStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.SkipStage;
-import de.bwaldvogel.mongo.backend.aggregation.stage.UnwindStage;
+import de.bwaldvogel.mongo.backend.aggregation.stage.*;
```

Comment:
Using wildcard import statements is not a good idea for several reasons. You can find some explanaitions here: https://stackoverflow.com/a/147461

Reply:
Agreed - the IDE ended up doing that, didn't notice - thanks.

Reply:
If you use IDEA or Eclipse, you could try to import the code style from [/code-style](https://github.com/bwaldvogel/mongo-java-server/tree/master/code-style)